.eslintrc

```diff
@@ -31,6 +31,12 @@
         "caughtErrorsIgnorePattern": "^_"
       }
     ],
+    "node/no-extraneous-import": [
```

Comment:
Är inte detta fel lösning på problemet? Är inte rätt lösning att lägga in http-errors i package.json som dependency (den finns bara i package-lock.json)

Reply:
Låter rimligt.. testar det

Reply:
Funkade bra!